backend/db/migrations/03-create-booking.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+let options = {};
```

Comment:
You should be specifying the schema when the environment is set to production. This should work in development (local), but it will break when you try to deploy